core/amber/src/main/resources/python_udf/demo_non_map_udf.py

```diff
@@ -1,21 +1,29 @@
+import logging
+
 import texera_udf_operator_base
 
 
 class DemoOperator(texera_udf_operator_base.TexeraUDFOperator):
+    logger = logging.getLogger("PythonUDF.DemoOperator")
+
     def __init__(self):
```

Comment:
why arent we putting the decorator on init?

Reply:
added.